src/icos_config/forever.js

```diff
@@ -0,0 +1,77 @@
+import { toPromise, formatNumber } from '../utils';
+import { convertWeb3Value } from '../utils/web3';
+
+export default {
+  crowdSaleTokenContract: '0xFB072e58741738fC26E86D2c49db3179Ea8d5D55',
+  tokenContract: '0xFDb1D263112b0A668dCe5748d961B4851Cfca8Ee',
+  baseCurrency: 'ETH',
+  information: {
+    name: 'Forever Has Fallen',
+    website: 'https://foreverhasfallen.com/',
+    logo: 'https://foreverhasfallen.com/logo/fhf-icon-only.png',
+  },
+  events: {
+    TokenSale: {
+      args: {
+        tokens: 'tokenAmount',
+        ether: 'etherAmount',
+        sender: 'tokenReceiver',
+      },
+      customArgs: {
+        tokenPrice: 'tokensPerEther', // Information on token price
```

Comment:
`customArgs` are used when you want to track special value that send with certain log, for example some people are using `Transfer` which is general event and the goal is to track issuing token which has `from: 0x000` for example, then you can use custom args.

Please remove `customArgs` or use it properly.